sass/sass.bzl

```diff
@@ -11,151 +11,114 @@
+              else output_name)
+  outputs = {
+      "css_file": css_file,
+      "map_file": "%s.map" % css_file,
+  }
+
+  return outputs
+
 sass_deps_attr = attr.label_list(
-    providers = ["transitive_sass_files"],
+    doc = "sass_library targets to include in the compilation",
+    providers = [SassInfo],
     allow_files = False,
 )
 
 sass_library = rule(
+    implementation = _sass_library_impl,
     attrs = {
         "srcs": attr.label_list(
-            allow_files = SASS_FILETYPES,
+            doc = "Sass source files",
+            allow_files = _FILETYPES,
             non_empty = True,
             mandatory = True,
         ),
         "deps": sass_deps_attr,
     },
-    implementation = _sass_library_impl,
-)
-
-sass_binary = rule(
-    attrs = {
-        "src": attr.label(
-            allow_files = SASS_FILETYPES,
-            mandatory = True,
-            single_file = True,
-        ),
-        "output_style": attr.string(default = "compressed"),
-        "deps": sass_deps_attr,
-        "_sassc": attr.label(
-            default = Label("//sass:sassc"),
-            executable = True,
-            cfg = "host",
-            single_file = True,
-        ),
-    },
-    outputs = {
-        "css_file": "%{name}.css",
-        "css_map_file": "%{name}.css.map",
-    },
-    implementation = _sass_binary_impl,
-)
-
-LIBSASS_BUILD_FILE = """
-package(default_visibility = ["@sassc//:__pkg__"])
-
-filegroup(
-    name = "srcs",
-    srcs = glob([
-         "src/**/*.h*",
-         "src/**/*.c*",
-    ]),
-)
-
-# Includes directive may seem unnecessary here, but its needed for the weird
-# interplay between libsass/sassc projects. This is intentional.
-cc_library(
-    name = "headers",
-    includes = ["include"],
-    hdrs = glob(["include/**/*.h"]),
 )
+"""Defines a group of Sass include files.
 """
 
-SASSC_BUILD_FILE = """
-package(default_visibility = ["//visibility:public"])
-
-cc_binary(
-    name = "sassc",
-    srcs = [
-        "@libsass//:srcs",
-        "sassc.c",
-        "sassc_version.h",
-    ] + select({
-        "@bazel_tools//src/conditions:windows": glob([
-            "win/**/*.c",
-            "win/**/*.h",
-        ]),
-        "//conditions:default": [],
-    }),
-    includes = select({
-        "@bazel_tools//src/conditions:windows": ["win/posix"],
-        "//conditions:default": [],
-    }),
-    linkopts = select({
-        "@bazel_tools//src/conditions:windows": ["-DEFAULTLIB:shell32.lib"],
-        "//conditions:default": [
-            "-ldl",
-            "-lm",
+_sass_binary_attrs = {
+    "src": attr.label(
+        doc = "Sass source files",
+        allow_files = _FILETYPES,
+        mandatory = True,
+        single_file = True,
+    ),
+    "include_paths": attr.string_list(
+        doc = "Additional directories to search when resolving imports"),
+    "output_dir": attr.string(
+        doc = "Output directory, relative to xxx",
```

Comment:
I'd assume relative to the workspace root?

Reply:
seems like it's relative to this package